test_scripts/Protocol/NACKReason/014_Invalid_time_RPC_Service_NACK.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0308-protocol-nak-reason.md
+--
+-- Description: SDL provides reason information in NACK message
+-- in case NACK received because system time is not provided during service starting
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. Mobile app is registered with 'NAVIGATION' HMI type and with 5 protocol
+-- 3. Mobile app is activated
+--
+-- Steps:
+-- 1. Mobile app requests the opening of protected RPC service
+-- 2. SDL requests system time via BC.GetSystemTime request
+-- 3. HMI responds with DATA_NOT_AVAILABLE resultCode
+-- SDL does:
+-- - respond with NACK to StartService request because system time is not provided
+-- - provide reason information in NACK message
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require("test_scripts/Protocol/commonProtocol")
+
+--[[ Local Variables ]]
+local isPTUtriggered = true
+local rpcServiceParams = {
+  reqParams = {
+    protocolVersion = { type = common.bsonType.STRING, value = "5.3.0" }
+  },
+  nackParams = {
+    reason = { type = common.bsonType.STRING, value = "Failed to get system time" }
+  }
+}
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("ForceProtectedService = 0x0A, 0x0B", common.sdl.setSDLIniParameter,
+  { "ForceProtectedService", "0x0A, 0x0B" })
```

Comment:
@jacobkeeler , this is a required step for this test otherwise, SDL will start unsecured service after failed PTU and send ACK to StartService request.

Reply:
@GetmanetsIrina This test is for the RPC service, `0x0A` and `0x0B` refer to audio and video services, so they aren't relevant here.

Reply:
@jacobkeeler , done in cefa130